docs/content/doc/usage/labels.en-us.md

```diff
@@ -0,0 +1,38 @@
+---
+date: "2023-03-04T19:00:00+00:00"
+title: "Usage: Labels"
+slug: "labels"
+weight: 13
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Labels"
+    weight: 13
+    identifier: "labels"
+---
+
+# Labels
+
+Issues and pull requests can be assigned labels for organization.
```

Comment:
I've had to re-read the sentence thrice as it always sounded like broken English, but I think I get it now:
```suggestion
You can use labels to classify issues and pull requests and to improve your overview over them.
```